seed/views/v3/progress.py

```diff
@@ -0,0 +1,35 @@
+from django.http import JsonResponse
+from rest_framework import viewsets
+
+from seed.decorators import ajax_request_class
+from seed.utils.api import api_endpoint_class
+from seed.utils.cache import get_cache
+
+import logging
+_log = logging.getLogger(__name__)
+
+
+class ProgressViewSet(viewsets.ViewSet):
+    raise_exception = True
+
+    @api_endpoint_class
+    @ajax_request_class
+    def retrieve(self, request, pk):
```

Comment:
Is there an easy to authorize this view? It's probably not that important but figured I'd ask

Reply:
Discussed: low authorization impact and any solution would be out of scope here.